package.json

```diff
@@ -95,6 +100,7 @@
     "html-loader": "0.5.5",
     "html-webpack-harddisk-plugin": "1.0.1",
     "html-webpack-plugin": "4.0.0-beta.5",
+    "husky": "^2.4.1",
```

Comment:
Just for consistency and safely, using strict package is recommended, like `2.4.1`